### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Ranausama420**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by tianyizheng02** [comment]
How is this PR different from #8633?

**Comment by ChrisO345** [comment]
In order to merge a PR, all test must pass, can you try to fix these errors.

File: ciphers/mixed_keyword_cypher.py [hunk 1]
```diff
@@ -21,23 +30,30 @@ def mixed_keyword(key: str = "college", pt: str = "UNIVERSITY") -> str:
     """
     key = key.upper()
     pt = pt.upper()
-    temp = []
-    for i in key:
-        if i not in temp:
-            temp.append(i)
+    ## changed below lines added sets functionality and remove for loops to get unique elemnt in list
+    temp = list(set(key))
```
Comment (CaedenPH) [inline ciphers/mixed_keyword_cypher.py:34]: This is not a documentation change

File: ciphers/mixed_keyword_cypher.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
+import string
+
+"""This function takes two parameters: a string variable named
+    'key' (with a default value of 'college') and a string variable
+     named 'pt' (with a default value of 'UNIVERSITY').
+     The function returns a string that is a transformation of the 'pt' argument
+      based on a key-shift cipher"""
```
Comment (CaedenPH) [inline ciphers/mixed_keyword_cypher.py:7]: Firstly, this isn't inside of a function and isn't formatted
Secondly, what does this tell the user that we don't already know? Everything you said is clearly shown by typehints and naming conventions

File: ciphers/mixed_keyword_cypher.py [hunk 3]
```diff
@@ -21,23 +30,30 @@ def mixed_keyword(key: str = "college", pt: str = "UNIVERSITY") -> str:
...
-            temp.append(t)
+    for j in alpha:
+        if j not in temp:
+            temp.append(j)
+
     # print(temp)
     r = int(26 / 4)
-    # print(r)
+    print(r)
+    print(len_temp)
```
Comment (CaedenPH) [inline ciphers/mixed_keyword_cypher.py:47]: Remove these prints
```suggestion
```

File: ciphers/mixed_keyword_cypher.py [hunk 4]
```diff
@@ -21,23 +30,30 @@ def mixed_keyword(key: str = "college", pt: str = "UNIVERSITY") -> str:
...
+    # for i in range(r*len_temp):
+    #     s = [temp[i] for _ in range(len_temp) if i < 26]
+    #     modalpha.append(s)
+
```
Comment (CaedenPH) [inline ciphers/mixed_keyword_cypher.py:51]:
```suggestion
```
Along with this random temp code

File: ciphers/mixed_keyword_cypher.py [hunk 5]
```diff
@@ -21,23 +30,30 @@ def mixed_keyword(key: str = "college", pt: str = "UNIVERSITY") -> str:
...
     k = 0
+    """These lines of code creates a dictionary by iterating over the list of
+    lists. Each letter in the alphabets list is mapped to a letter in a row of
+     the "modalpha" list. The mappings are stored in the dictionary with the
+     indices of the alphabets list as keys and the values fromthe corresponding modalpha lists as values"""
```
Comment (CaedenPH) [inline ciphers/mixed_keyword_cypher.py:56]: These should be in the forms of comments, not docstrings

**Review by CaedenPH — CHANGES_REQUESTED** [review]
This change does not add any information that the person reading the code cannot infer themselves - Seems pointless

**Comment by tianyizheng02** [comment]
@CaedenPH this PR is basically identical to an earlier PR #8633 (same commits as this one), so one of the two should be closed. We can probably keep this PR since you already started a review on this one.

There's also an even earlier PR #8626 by a different author that fixes the same issue #8622, so it'd be good if you could take a look at that one as well.

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]
> This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.

Please make separate PRs for your changes to `mixed_keyword_cypher.py` and your changes to the other files

**Comment by tianyizheng02** [comment]
Closed as duplicate of #8633

**Pull request closed without merging**